roles/confluent.test/molecule/plain-rhel/molecule.yml

```diff
@@ -99,7 +99,7 @@ provisioner:
     group_vars:
       all:
         sasl_protocol: plain
-
+        kafka_broker_configure_control_plane_listener: true
```

Comment:
lets add some tasks to the verify.yml that checks the properties are there

Reply:
I've done it [here](https://github.com/confluentinc/cp-ansible/blob/fcf0bad0420a5a06d7dc11620144de7808ff6cee/roles/confluent.test/molecule/plain-rhel/verify.yml#L46). Would you like me to add some additional tests?